test/kubemark/resources/hollow-node_template.json

```diff
@@ -106,7 +106,7 @@
 				},
 				{
 					"name": "hollow-proxy",
-					"image": "gcr.io/{{project}}/kubemark:latest",
+					"image": "{{registry}}/{{project}}/kubemark:latest",
 					"env": [
 						{
 							"name": "CONTENT_TYPE",
```

Comment:
I'm not sure about this because node-problem-detector, just like other components (apiserver, scheduler, etc) is part of proper versioned releases. Unlike kubemark which we just build and push locally to the project on-the-go (as we want it to be built against the HEAD).